napalm_logs/config/opengear/LOGIN.yml

```
@@ -0,0 +1,13 @@
+#<30>ztp-tankstelle_1-ts port02 2018-Nov-12 10:47:29.515 LOGIN: root
+
+messages:
+  - error: LOGIN
```

```suggestion
  - error: USER_LOGIN
```

We already have this one defined - see https://napalm-logs.com/en/latest/messages/USER_LOGIN.html